test/integration/README.md

```diff
@@ -13,26 +13,26 @@ $ go test -v .
 First, run the generator to create `openapi_generated.go` file which specifies
 the `OpenAPIDefinition` for each type, and generate the golden API rule
 violation report file . Note that if you do not pass a report
-filename (`./testdata/golden.report` in the command below) to let the generator
+filename (`./testdata/golden.v2.report` in the command below) to let the generator
```

Comment:
Why are all of these v2 now?

Reply:
Oh I see, you're fixing the doc that is incorrect, thanks.

Reply:
👍